ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-commands.sh

```diff
@@ -215,11 +219,19 @@ wait_deploy central
 get_init_bundle
 install_secured_cluster_with_helm
 echo ">>> Wait for 'stackrox-secured-cluster-services' deployments"
+if [[ "${ROX_SCANNER_V4:-true}" == "true" ]]; then
+  wait_deploy scanner-v4-indexer
+  wait_deploy scanner-v4-matcher
```

Comment:
The Matcher, in particular, can only really serve if vulnerabilities are loaded. We are working towards allowing this [here](https://github.com/stackrox/stackrox/pull/14200) -- but we would probably need to increase the timeout for the Matcher to around 30m to accomodate vuln loading. Is that assumption correct?